src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -248,8 +248,8 @@ protected int loadWarehouse(Connection conn, int w_id) {
 			warehouse.w_street_1 = TPCCUtil.randomStr(TPCCUtil.randomNumber(10, 20, benchmark.rng()));
 			warehouse.w_street_2 = TPCCUtil.randomStr(TPCCUtil.randomNumber(10, 20, benchmark.rng()));
 			warehouse.w_city = TPCCUtil.randomStr(TPCCUtil.randomNumber(10, 20, benchmark.rng()));
-			warehouse.w_state = TPCCUtil.randomStr(3).toUpperCase();
-			warehouse.w_zip = "123456789";
+			warehouse.w_state = TPCCUtil.randomStr(2).toUpperCase();
+			warehouse.w_zip = TPCCUtil.randomNStr(4) + "11111";
```

Comment:
Please add this to the description as well - since your diff includes more than just the changes for think/keying time.

Reply:
Done.

Reply:
The comment already had these details/. {point 3 in the message above}